### ISSUE TITLE
About Loss

### ISSUE THREAD

**Issue by TohnoYouki** [stonyhu/DanceRevolution#10]
Can you tell me how much the loss of the test set is after training? I think my loss is too large

**Comment by stonyhu** [comment]
You do not need to care about the loss of the test set, it cannot reflect the real performance on the test set. If the L1 loss of test set is very low, it means that the generated sequences are very similar to the ground-truth. However, what we want is to generate the pose sequences that are different from the ground-truth sequences. In this case, the L1 loss would be high.

**Comment by TohnoYouki** [comment]
@stonyhu Thank you for reply! But how to detect the overfitting in that case? In fact, I am troubled by the problem that the trained model performs well on train set, but when using the test set, the amplitude of the dance(or richness of the dance) is not so well. Could you give me some advise for training the model or avoiding this problem? Thank you very much!

**Comment by stonyhu** [comment]
@TohnoYouki You could have a try to generate multiple dance sequences for each music clip from test set and choose the one you think is the best. In the dance decoder, the initial hidden state is initialized by sampling from standard normal distribution. Hence, the model have the ability to generate the different dance pose sequences for one music clip.

**Comment by stonyhu** [comment]
@TohnoYouki Hi, we have released the inference pipeline and you can refer to it.
